druid/0.15/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM openjdk:8-jre
+
+ENV DRUID_HOME=/usr/local/druid
+ENV DRUID_VERSION=0.15.0-incubating
+
+# perl needed for running some druid scripts
+RUN apt-get update \
+    && apt-get install -y perl \
+    && apt-get clean all && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p $DRUID_HOME \
+    && curl "https://archive.apache.org/dist/incubator/druid/$DRUID_VERSION/apache-druid-$DRUID_VERSION-bin.tar.gz" \
+    | tar zxf - --strip-components 1 -C $DRUID_HOME
+
+COPY ./conf /opt/druid_conf
+
+COPY docker-entrypoint.sh /
+RUN chmod +x /docker-entrypoint.sh
```

Comment:
We can reduce the number of layers by having a single RUN command instead of three.

Reply:
Hum don't really agree on this, by putting everything together it might be more difficult to read and maintain.

It common to separate directive by groups of commands that functionally go together.

It also common to organise directives from the ones that change the least frequently to the ones that change more often. To avoid rebuilding upper layer, especially useful when developing.

Reply:
The code on this repo does not need much maintenance but the build image will be pulled very often for running tests in pipelines. So I think this is typically one of the real use cases where it does make sense to trade maintainability for reducing the size of the image.

However, the impact is probably very small anyway

Reply:
![image](https://user-images.githubusercontent.com/49917914/64711285-a42d9600-d4b9-11e9-9dfb-43deb49e53ff.png)